code/src/css components/button1.css

```diff
@@ -0,0 +1,16 @@
+.submitbutton {
+  background: rgb(248, 196, 204);
+  color: grey;
+  border-style: outset;
+  border-color: white;
+  height: 50px;
+  width: 100px;
+  font: bold 15px arial, sans-serif;
+  text-shadow: none;
+}
+.submitbutton:hover {
+  background: rgb(248, 196, 204);
+}
+.submitbutton:focus {
+  outline: thin dotted;
+}
```

Comment:
My suggestion here is to make 1  CSS-file for "Buttons" and link the same file to each component that uses it. Then you don't have to have repeated code in many files – which makes it hard to control if you want to make changes.